panel/widgets/tables.py

```diff
@@ -896,6 +896,29 @@ def _update_aggregators(self, model):
             g.aggregators = self._get_aggregators(index)
 
 
+class _ListValidateWithCallable(param.List):
```

Comment:
If we are defining custom parameters (which we should do much more of) I think we should put them in a shared place.

Reply:
The logical place will be `panel/param.py`, but the file could get pretty large if a lot of parameters are added.  

A small thing is the current error message is tailored to make sense with `Tabulator` only.

Reply:
I'd love for `param` to provide a way to add validators that'd be called before the new value is set.

Reply:
Agreed, I made an issue for that years ago https://github.com/holoviz/param/issues/324